bundles/org.openhab.io.homekit/README.md

```diff
@@ -828,8 +877,36 @@ or using UI
+|                      |                             | ConfiguredName               | String                        | Name of the input source configured in the iOS Home app. User can rename the source in iOS Home app and this characteristic can be used to reflect change in openHAB and sync name changes from openHAB to Home app. |
+|                      |                             | Configured                   | Switch, Contact, Dimmer       | If the source is configured on the device. Non-configured inputs will not show up in the Home app. - ON/OPEN = show, OFF/CLOSED = hide. Default is ON. Can also be configured via metadata, e.g. [Configured=true] |
+|                      |                             | InputSourceType              | String                        | Type of the input source. possible values (OTHER, HOME_SCREEN, TUNER, HDMI, COMPOSITE_VIDEO, S_VIDEO, COMPONENT_VIDEO, DVI, AIRPLAY, USB, APPLICATION). Custom mapping can be defined at item level. Can also be configured via metadata, e.g. [InputSourceType="OTHER"]. |
+|                      |                             | CurrentVisibility            | Switch, Contact, Dimmer       | If the source has been hidden by the user - ON/OPEN = visible, OFF/CLOSED = hidden. Default is ON. Can also be configured via metadata, e.g. [CurrentVisibility=false] |
+|                      |                             | Identifier                   | Number                        | The identifier of the source, to be used with the ActiveIdentifier characteristic. Can also be configured via metadata, e.g. [Identifier=1] |
+|                      |                             | InputDeviceType              | String                        | Type of the input device. possible values (OTHER, TV, RECORDING, TUNER, PLAYBACK, AUDIO_SYSTEM). Custom mapping can be defined at item level. Can also be configured via metadata, e.g. [InputDeviceType="OTHER"]. |
+|                      |                             | TargetVisibilityState        | Switch                        | The desired visibility state of the input source. ON = shown, OFF = hidden. |
+| TelevisionSpeaker    |                             |                              |                               | An accessory that can be added to a Television in order to control the speaker associated with it. |
+|                      | Mute                        |                              | Switch                        | If the television is muted. ON = muted, OFF = not muted. |
+|                      |                             | Active                       | Switch                        | ?? |
```

Comment:
No description?

Reply:
Nope. No idea what it does. TV Accessory isn't documented at all in the public documentation. A lot of this is based off of reverse engineering the HAP-JS team has done. I couldn't find that this characteristic had any effect.

Reply:
Hmm, it looks strange documenting it with "??". But leaving it out when it actually exists would leave it even more undocumented. Perhaps add the text "<undocumented>" or something like that rather than "??". That would at least clearly state that it's not a "TODO" kind of left-over, but rather something which is not possible to document at the moment.

Reply:
Will do.